prettyprinter-configurable/prettyprinter-configurable.cabal

```diff
@@ -82,7 +82,7 @@ test-suite prettyprinter-configurable-test
     , base                        >=4.9 && <5
     , megaparsec
     , parser-combinators
-    , prettyprinter-configurable  ^>=1.6
+    , prettyprinter-configurable
```

Comment:
This shouldn't be changed.

Reply:
Cabal says that is a redundant constraint because, the test suite will always use the library defined in the cabal file.

Reply:
Ah I see, I missed where it was, sorry!